shared/packages/worker/src/worker/workers/windowsWorker/expectationHandlers/lib/ffmpeg.ts

```diff
@@ -25,20 +25,20 @@ export function testFFProbe(): Promise<string | null> {
 }
 export function testFFExecutable(ffExecutable: string): Promise<string | null> {
 	return new Promise<string | null>((resolve) => {
-		const ffMpegProcess: ChildProcess = spawn(ffExecutable, ['-version'])
+		const ffMpegProcess = spawn(ffExecutable, ['-version'])
 		let output = ''
-		ffMpegProcess.stderr?.on('data', (data) => {
+		ffMpegProcess.stderr.on('data', (data) => {
 			const str = data.toString()
 			output += str
 		})
-		ffMpegProcess.stdout?.on('data', (data) => {
+		ffMpegProcess.stdout.on('data', (data) => {
 			const str = data.toString()
 			output += str
 		})
 		ffMpegProcess.on('error', (err) => {
 			resolve(`Process ${ffExecutable} emitted error: ${stringifyError(err)}`)
 		})
-		ffMpegProcess.on('close', (code) => {
+		ffMpegProcess.on('exit', (code) => {
```

Comment:
Out of curiosity, what's your reasoning for using `exit` instead of `close` here?

Reply:
Multiple processes can be attached to the same STDIO streams. `close` will be emitted whenever any of those processes closes the stream. `exit` will be emitted when the child_process in question exits: https://nodejs.org/docs/latest-v12.x/api/child_process.html#child_process_event_close